commcare_connect/opportunity/migrations/0110_paymentinvoice_status_modified_date.py

```
@@ -0,0 +1,17 @@
+# Generated by Django 4.2.5 on 2026-02-06 07:05
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("opportunity", "0109_paymentunit_org_amount"),
+    ]
```

Ruff (0.14.14)

[warning] Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)